projman_filler/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 """Top-level package for ProjMan Filler."""
 
-__version__ = '1.6.3'
+__version__ = '1.6.4'
```

Comment:
I wonder if it should be at least a minor version update since the python requirement have changed.